### ISSUE TITLE
provide an official supported way to get large files from a user’s computer into CoCalc

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#2223]
This is a recurring problem.  I don’t know what approach is best.  Ideas are welcome.

IDEAS:
- provide a wiki page with links to software for windows, etc., explaining how to split a file.
- find some actual program that automates the process for users
- do something clever involving google drive, etc. 
- ????

Basically, we need a canonical answer for non-tech-savvy users.

**Comment by johnjeng** [comment]
Yes. But wondering how large is large? A size definition? A functional one like size based on chance of interruption on a "standard" internet connection?

**Comment by haraldschilly** [comment]
From what I've seen so far, everything above 50M is "risky" (i.e. it did fail for me once when I tried it).

Also, more general, such a wiki page should explain that several files can be selected and compressed in a  `.zip` file.

**Comment by williamstein** [comment]
Any size.  It’s 100% free to us by the way.  There is zero charge for incoming data to gce.

**Comment by williamstein** [comment]
The motivating problem was a support request for a 650mb file.

**Comment by williamstein** [comment]
Now that we have an excellent SSH gateway with very nice ssh key support, the general solution should be “use rsync and ssh”.  Still, we need some better documentation about this.

**Comment by williamstein** [comment]
I also found that there was a default 125MB limit set by cloudflare.  I’ve changed it to the max they allow, which is 200MB.
